src/main/java/com/baidu/hugegraph/api/auth/ProjectAPI.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2017 HugeGraph Authors
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.baidu.hugegraph.api.auth;
+
+import java.util.List;
+import java.util.Map;
+
+import com.baidu.hugegraph.client.RestClient;
+import com.baidu.hugegraph.rest.RestResult;
+import com.baidu.hugegraph.structure.auth.Project;
+import com.baidu.hugegraph.structure.constant.HugeType;
+import com.google.common.collect.ImmutableMap;
+
+public class ProjectAPI extends AuthAPI {
+
+    private static final String ACTION_ADD_GRAPH = "add_graph";
+    private static final String ACTION_DELETE_GRAPH = "delete_graph";
```

Comment:
remove_graph

Reply:
i suggest don't remove '_graph', because "action" as a params of update project, maybe "add_graph" is more intuitive than "add", maybe we will add "add_a" or "add_b" in the future

Reply:
use "remove_graph" instead of "delete_graph"